docs/modules/ROOT/pages/vshn-managed/nextcloud/database.adoc

```diff
@@ -0,0 +1,75 @@
+= Managed Database
+
+There are two ways to configure a managed database for the VSHNNextcloud service.
+
+== Dedicated VSHNPostgreSQL Service
+
+In this setup, each `VSHNNextcloud` instance gets its own dedicated `VSHNPostgreSQL` service.
```

Comment:
I'd probably note somewhere that this is the default.

Reply:
done